pkg/geo/s2.go

```diff
@@ -119,11 +119,17 @@ func Covering(points []s2.Point) (s2.CellUnion, error) {
 // CoveringForLoop calculates an s2 cell covering for loop or returns an area if
 // the area covered by loop is too large.
 func CoveringForLoop(loop *s2.Loop) (s2.CellUnion, error) {
-	if loopAreaKm2(loop) > maxAllowedAreaKm2 {
```

Comment:
might make more sense to clean this up in the above `Covering` function, ie:

```
func Covering(points []s2.Point) (s2.CellUnion, error) {
  loop := s2.LoopFromPoints(points)
  area := loop.Area() // change to area
  if areaTooLarge(area) {
    loop.Invert()
  }
 if areaTooLarge(area) {
   return nil, ErrTooLarge
 }
  if area <= 0 {
      pl := s2.Polyline(loop.Vertices())
		  return RegionCoverer.Covering(&pl), nil
   }
  return RegionCoverer.Covering(loop), nil
}
```

Reply:
Done, and corrected a problem from #295 as well.